minimisers/generalUtils/makeEmptyBayesResultsStruct.m

```diff
@@ -2,29 +2,31 @@
     % A function to make an empty container to hold the results of bayes
     % calculations. The struct has the following format:
     %
-    % nPar = number of fitted parameters
+    % nParams = number of fitted parameters
     % nContrasts = number of contrasts
     %
     % bayesResults = 
     % 
     %   struct with fields:
     % 
-    %     bestFitsMean: [1×1 struct]
-    %         predlims: [1×1 struct]
-    %      parConfInts: [1×1 struct]
-    %         bestPars: [1xnPars double]
-    %         bayesRes: [1×1 struct]
-    %            chain: [1000000xnPars double]
+    %         bestFitMean: [1×1 struct]
+    % predictionIntervals: [1×1 struct]
+    % confidenceIntervals: [1×1 struct]
+    %           allChains: [1×1 struct]
```

Comment:
any relation between chain and allChains?

Reply:
Hmm, they're very large numerical arrays of different sizes and shapes. Might be worth asking about this one before doing anything.